aikau/src/main/resources/alfresco/forms/controls/DojoRadioButtons.js

```diff
@@ -23,6 +23,4 @@
  * @author Dave Draper
  * @deprecated Since 1.0.3 - Use [alfresco/forms/controls/RadioButtons]{@link module:alfresco/forms/controls/RadioButtons} instead
  */
-define(["dojo/_base/declare",
-        "alfresco/forms/controls/RadioButtons"], 
-        function(declare, RadioButtons) {});
\ No newline at end of file
+define([], function() {});
```

Comment:
The callback supplied to the define function is empty, therefore there's no point in loading either declare or RadioButtons into the closure for that function. It's simply a removal of redundant code.